claims.go

```diff
@@ -37,7 +37,7 @@ func (c StandardClaims) Valid() error {
 	// default value in Go, let's not fail the verification for them.
 	if c.VerifyExpiresAt(now, false) == false {
 		delta := time.Unix(now, 0).Sub(time.Unix(c.ExpiresAt, 0))
-		vErr.Inner = fmt.Errorf("token is expired by %v", delta)
+		vErr.Inner = fmt.Errorf("Token is expired by %v", delta)
```

Comment:
Ok, it looks like the Go way is to make a new error type for and convey the structured data that way. The message can remain the same, preserving backward compatibility.